src/layers/conv.jl

```diff
@@ -244,8 +267,24 @@ _channels_out(l::ConvTranspose) = size(l.weight)[end-1]*l.groups
 """
     ConvTranspose(weight::AbstractArray, [bias, activation; stride, pad, dilation, groups])
 
-Constructs a layer with the given weight and bias arrays.
-Accepts the same keywords as the `ConvTranspose((4,4), 3 => 7, relu)` method.
+Constructs a ConvTranspose layer with the given weight and bias.
+Accepts the same keywords and has the same defaults as `ConvTranspose((4,4), 3 => 7, relu)`.
```

Comment:
```suggestion
Accepts the same keywords and has the same defaults as `ConvTranspose(k::NTuple{N,Integer}, ch::Pair{<:Integer,<:Integer}, σ; ...)`.
```
I'm not sure if `@ref` works here, but if it does I would recommend using it to refer to the other constructor.